assets/src/application/ui/layout/entityList/EntityList.tsx

```diff
@@ -56,7 +57,12 @@ const EntityList = <E extends Entity, ELFS extends EntityListFilterStateManager<
 			<Title className='ee-entity-list__header' level={3}>
 				{headerText}
 			</Title>
-			<EntityListFilterBar domain={domain} filterState={filterState} listId={listId} />
+			<EntityListFilterBar
+				domain={domain}
+				filterState={filterState}
+				legendConfig={legendConfig}
```

Comment:
I didn't like this at first because I thought these were going to display along with the filters but after checking out the branch and trying this I really like that this keeps the UI cleaner and has all of the buttons in the same place. 👍